elastic/kibana
# [ObsUX] Performance adjustments on the Inventory UI

closes: https://github.com/elastic/kibana/issues/170508

## Summary

This PR makes a small adjustment in the inventory UI, optimizing the usage of the Snapshot API.

Basically, 2 things were changed:

1 - AbortController, to interrupt ongoing any requests when new ones are made
2 - Stop requesting time-series data in the waffle map view (table view and history section still need it)


### `/snapshot` with time-series data
<img width="1400" alt="image" src="https://github.com/elastic/kibana/assets/2767137/562aaaf0-299d-4170-87e2-0e92b309f50c">

<img width="1400" alt="image" src="https://github.com/elastic/kibana/assets/2767137/29d68239-17e6-416f-be7b-c5928cf1545c">


<img width="1400" alt="image" src="https://github.com/elastic/kibana/assets/2767137/619473b8-983a-4ba7-a279-b44b5f2bbfbd">

### `/snapshot` without time-series data
<img width="1400" alt="image" src="https://github.com/elastic/kibana/assets/2767137/4f25caf9-08cd-4388-8a54-7916a3c70786">

<img width="1400" alt="image" src="https://github.com/elastic/kibana/assets/2767137/5719d012-57ce-42aa-a186-f65827b487e0">

<img width="1400" alt="image" src="https://github.com/elastic/kibana/assets/2767137/2df40aed-8a5f-4428-af85-34997789f62a">





### How to test
- Navigate to `Infrastructure`
- Inspect the `/snapshot` request to see if time-series data was returned
  - it should be returned when: opening the History accordion  and loading table view
  - it shouldn't be returned when: hovering on a waffle map item, loading waffle map view
- Ongoing requests must be aborted when new ones are made
